sdk/storage/inc/blobs/unused/BlobServiceClient.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <chrono>
+
+#include "azure/core/Pipeline.h"
+#include "azure/core/Uri.h"
+
+#include "azure/storage/common/Lease.h"
+#include "azure/storage/common/StorageSharedKeyCredential.h"
+
+#include "models/BlobClientOptions.h"
+#include "models/CreateBlobContainerOptions.h"
+#include "models/DeleteBlobContainerOptions.h"
+#include "models/GetUserDelegationKeyOptions.h"
+#include "models/ListBlobContainersOptions.h"
+
+namespace Azure { namespace Storage {
+
+class BlobServiceClient
+{
+public:
+    // Connection string
+    BlobServiceClient(const std::string& connectionString, BlobClientOptions options = BlobClientOptions());
```

Comment:
explicit

Reply:
Agreed. We should talk about adding the explicit guidance at our next meeting.